spec/models/vote_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Vote do
+  let(:user) { create(:user) }
```

Comment:
`user` and `event` are not needed. `let(:vote) { create(:vote) }` will create them. (That's why we use factories, to save all this work :wink: )

Reply:
@kaustubh-nair would you please take care of the review comments, so that we can merge this?

Reply:
Oh sorry @differentreality 
I'll make the changes and push again asap